lib/rules/jsx-sort-props.js

```diff
@@ -249,6 +249,9 @@ module.exports = {
         },
         reservedFirst: {
           type: ['array', 'boolean']
+        },
+        ignoreTags: {
+          type: 'array'
```

Comment:
this would need to ensure the items are unique, and are valid custom component names

Reply:
Agree as for the uniqueness. Not sure about validity of component names - see my comment above.